backend/api/Areas/Property/Controllers/PropertyController.cs

```diff
@@ -112,15 +112,15 @@ public IActionResult GetConceptPropertyWithPid(string pid)
         }
 
         /// <summary>
-        /// Update the specified contact, and attached properties.
+        /// Update the specified property, and attached properties.
         /// </summary>
         /// <returns></returns>
         [HttpPut("concept/{pid}")]
         [HasPermission(Permissions.PropertyEdit)]
         [Produces("application/json")]
         [ProducesResponseType(typeof(Pims.Api.Models.Concepts.PropertyModel), 200)]
         [SwaggerOperation(Tags = new[] { "property" })]
-        public IActionResult UpdatePerson([FromBody] Pims.Api.Models.Concepts.PropertyModel propertyModel)
+        public IActionResult UpdateConceptProperty([FromBody] Pims.Api.Models.Concepts.PropertyModel propertyModel)
```

Comment:
Why the change of name?

Reply:
It was a copy/paste mistake from previous PR - this is the property controller, not the person controller